packages/@lightningjs/ui-components/src/docs/ThemingTutorial.stories.mdx

```diff
@@ -0,0 +1,267 @@
+        }
+    }
+
+    _getFocused() {
+        return this.tag('Page');
+    }
+}
+```
+
+## Creating a Page
+
+### Understanding the Base Component
+
+All components in LUI extend a single `Base` component, which you can read about [here]([Theming Overview](../?path=/story/docs-base-component--page). This Base component is wrapped in helpful mixins like [`withThemeStyles`](../?path=/docs/utilities-withthemestyles--with-theme-styles), [`withHandleKey`](../?path=/docs/utilities-withhandlekey--with-handle-key), [`withUpdates`](../?path=/docs/utilities-withupdates--with-updates), and [`withTags`](../?path=/docs/utilities-withtags--with-tags). It is essential to utilize this Base component in your applications in order to get the full benefits of theming.
+
+We will start building our custom `ExamplePage` by extending this `Base` component, like so:
+
+```js
+import { Base } from '@lightningjs/ui-components';
+
+export default class ExamplePage extends Base {
+  static get __componentName() {
+    return 'ExamplePage';
+  }
+}
+```
+
+### Simplifying the static _template
+
+While it is common to set up most of a component in the `static _template`, you will not be able to get the full benefits of a custom theme by using the template alone. This is because our components have a unique `update` lifecycle (via [`withUpdates`](../?path=/docs/utilities-withupdates--with-updates)) that ensures any time a style property is update (whether from a global theme update or a change to an instance of a component), they re-render.
+
+If you reference theme properties inside of the `_template`, which only runs *once*, regardless of theme updates, your custom component will never render with the new custom theme values. For example, if you have a custom theme set for your application, things will run in the following order:
+- Application startup
+- Component creation utilizing the Base theme
+- A new theme is set via the global `context`
+- The component receives an event to update its style
+
+In order to ensure the component always applies the most up-to-date style values, we will keep them out of the template:
+
+ExamplePage.js
+```js
+import { Base, Button, TextBox } from '@lightningjs/ui-components';
+
+export default class ExamplePage extends Base {
+  static get __componentName() {
+    return 'ExamplePage';
+  }
+
+  static _template() {
+    return {
+      Content: {
+        w: w => w,
+        h: h => h,
+        flex: {
+          direction: 'column',
+          justifyContent: 'center'
+        },
+        Title: { type: TextBox },
+        Description: { type: TextBox },
+        Button: { type: Button }
+      }
+    }
+  }
+}
+```
+
+### Leveraging withTags
+
+In order to make our developer lives easier as we write code, we will also add this static `tags` method to create local variable shorthands. For example, instead of `this.tag('Title')`, we can use `this._Title`. For more information, check the `withTags` documentation.
```

Comment:
"developer" -> "developers'"

Reply:
This was actually intentional, like talking about yourself, so should perhaps be hyphenated "develop-lives," but instead I just simplified to "make our lives"